corehq/apps/case_search/models.py

```diff
@@ -17,9 +12,11 @@
 CASE_SEARCH_BLACKLISTED_OWNER_ID_KEY = 'commcare_blacklisted_owner_ids'
 CASE_SEARCH_XPATH_QUERY_KEY = '_xpath_query'
 CONFIG_KEY_PREFIX = "x_commcare_"
+CASE_SEARCH_CASE_TYPE_KEY = "case_type"
```

Comment:
Can you explain what `CONFIG_KEY_PREFIX` is for, and why this one doesn't have it if it is a CONFIG KEY?